docs/topics/OAuth2.md

```diff
@@ -61,7 +61,7 @@ The authorization code grant is what most developers will recognize as "standard
 ###### Authorization URL Example
 
 ```
-https://discord.com/api/oauth2/authorize?response_type=code&client_id=157730590492196864&scope=identify%20guilds.join&state=15773059ghq9183habn&redirect_uri=https%3A%2F%2Fnicememe.website&prompt=consent
+https://discord.com/api/oauth2/authorize?response_type=code&client_id=157730590492196864&scope=identify%20guilds.join&state=15773059ghq9183habn&redirect_uri=https%3A%2F%2Fredirect.com&prompt=consent
```

Comment:
Should revert this too

Reply:
I think `example.com/redirect` is more fitting, as @BrightSkyz has said that this is just a ad for a advertising company.